astropy/config/tests/test_configs.py

```diff
@@ -10,7 +10,7 @@
 
 from ...tests.helper import catch_warnings
 from ...extern import six
-from ...utils.compat import subprocess
+import subprocess
```

Comment:
We usually follow the convention to have built-in modules at the top, so can you move this line to just under ``import sys``?